.github/workflows/trigger_testing.yaml

```
@@ -0,0 +1,106 @@
+name: Trigger testcases run workflow dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      pytest_marker:
+        description: 'Marker for specifying tests to run'
+        required: false
+        default: ''
+      region:
+        type: choice
+        description: 'Select the region'
+        required: false
+        default: 'NG'
+        options:
+        - "NG"
+        - "UG"
+      environment:
+        type: choice
+        description: 'Select the environment'
+        required: false
+        default: 'UAT'
+        options:
+        - 'UAT'
+        - 'PRODUCTION'
+      region_node:
+        description: 'Select the region node'
+        required: false
+        default: ''
+
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.2
+
+
+      - name: Install OpenVPN
+        run: |
+          sudo apt update
+          sudo apt install -y openvpn openvpn-systemd-resolved
```

again I'm not 100% sure about the path but I think this should work 
```suggestion
      - name: Cache OpenVPN installation  
        uses: actions/cache@v4  
        with:  
          path: /var/cache/apt/archives  
          key: ${{ runner.os }}-openvpn-${{ hashFiles('**/apt.lock') }}  
          restore-keys: |  
            ${{ runner.os }}-openvpn-  

      - name: Install OpenVPN  
        run: |  
          sudo apt update  
          sudo apt install -y openvpn openvpn-systemd-resolved  
      - name: Install OpenVPN
        run: |
          sudo apt update
          sudo apt install -y openvpn openvpn-systemd-resolved
```